stories/Components/MegaMenu/MegaMenu.jsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from "react";
+import "./megamenu.scss"
+
+export function MegaMenu() {
+  const sections = [
+    {
+      title: 'Section 1',
+      bannerHeading: 'Analytics by country',
+      bannerDescription: 'Gaze upon statistics in wonder...',
+      items: [
+        {
+          title: 'Item 1',
+          subItems: Array(30).fill({ title: 'Sub-item 1', url: '#'})
+        },
+        {
+          title: 'Item 2',
+          subItems: Array(20).fill({ title: 'Sub-item 2', url: '#'})
+        }
+      ]
+    },
+  ]
+
+  const [section, setSection] = useState(null);
+  const [itemIndex, setItemIndex] = useState(0);
+
+  return (
+    <div 
+      className="mg-mega-wrapper"
+      // onMouseLeave={() => setsection(null)}
+    >
+      {/* Topbar */}
+      <div className="mg-mega-topbar">
+        {
+          sections.map((item, index) => (
+            <div 
+              key={index}
+              className="mg-mega-topbar__item"
+              onMouseEnter={() => setSection(item)}
+            > 
+              { item.title }
+            </div>
+          ))
+        }
+      </div>
+      {/* Content */}
+      {
+        section !== null && (
+          <div className="mg-mega-content">
+            <div className="mg-mega-content__left">
+              <div className="mg-mega-content__banner">
+                <h1>{section.bannerHeading}</h1>
```

Comment:
Using H1s will get us into semantic and accessibility trouble; suggest we:

- swap the h1 to a `<header>` element
- make ` <div className="mg-mega-content">`  into ` <article className="mg-mega-content">`
- make `<div class="mg-mega-content__left">` into `<aside class="mg-mega-content__aside">`
- make `<div className="mg-mega-content__right">` into `<section className="mg-mega-content">`
- and `<div className="mg-mega-wrapper">` to `<nav className="mg-mega-wrapper">`

I think this will win us some better semantic-ness and make code a bit easier to read.